opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -140,3 +140,147 @@ const fn severity_of_level(level: &Level) -> Severity {
         Level::ERROR => Severity::Error,
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::layer;
+    use opentelemetry::logs::Severity;
+    use opentelemetry::trace::TracerProvider as _;
+    use opentelemetry::trace::{SpanId, TraceContextExt, TraceFlags, TraceId, Tracer};
+    use opentelemetry::{logs::AnyValue, Key};
+    use opentelemetry_sdk::logs::LoggerProvider;
+    use opentelemetry_sdk::testing::logs::InMemoryLogsExporter;
+    use opentelemetry_sdk::trace::{config, Sampler, TracerProvider};
+    use tracing::error;
+    use tracing_subscriber::layer::SubscriberExt;
+
+    // cargo test --features=testing
+    #[test]
+    fn tracing_appender_standalone() {
+        // Arrange
+        let exporter: InMemoryLogsExporter = InMemoryLogsExporter::default();
+        let logger_provider = LoggerProvider::builder()
+            .with_simple_exporter(exporter.clone())
+            .build();
+
+        let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
+        let subscriber = tracing_subscriber::registry().with(layer);
```

Comment:
nit - above 4 lines can be moved to a common function, which returns a pair of `subscriber` and `logger_provider`. Probably something we can do when we add more tests.

Reply:
yea holding off from writing helpers until we add more scenarios.